src/components/pages/pricing/faq/faq.jsx

```diff
@@ -13,16 +13,16 @@ const items = [
     answer: `Neon bills for the past month's usage at the beginning of each calendar month. For more information, see <a href="/docs/introduction/manage-billing">Manage billing</a>.`,
   },
   {
-    question: 'What is a compute-hour?',
-    answer: `Compute-hours is the metric for compute usage in Neon. It is short for “compute capacity consumed in a given hour”. The quick math: Compute-hours = CPU used x active time.`,
+    question: 'What is a compute hour?',
+    answer: `Compute hour is the metric for compute usage in Neon. The quick math: [compute hours] = [CPU size] x [hours your compute runs]. Each pricing plan includes a certain number of compute hours; how quickly you consume these hours depends on the size of your workload and how long it runs. Once you exceed the allocated amount of compute hours, you'll be billed for overages in a given month.`,
```

Comment:
Should "CPU size" be "compute size" or "CU", as per the Slack thread?

"how quickly you consume these hours depends on the size of your **compute** and how long it runs" ?